ihatemoney/templates/password_reminder.fr

```diff
@@ -1,7 +1,7 @@
 Salut,
 
-Vous avez demandez des informations sur votre mot de passe pour "{{ project.name }}".
+Vous avez demandé à réinitialiser votre mot de passe pour "{{ project.name }}". Ce lien est seulement valide pendant30 minutes.
```

Comment:
missing a space here!

Reply:
In the french and english version, I believe the `it's only valid for X minutes` should be *after* the actual link.